schemas/workflow.json

```diff
@@ -0,0 +1,142 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "definitions": {
+        "url": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "minLength": 1,
+                    "pattern": "^https?://.+"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "nullable-string": {
+            "oneOf": [
+                {
+                    "type": "string",
+                    "minLength": 1
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        }
+    },
+    "title": "workflow",
+    "description": "A workflow",
+    "type": "object",
+    "required": [
+        "name",
+        "command"
+    ],
+    "properties": {
+        "name": {
+            "title": "name",
+            "description": "A name of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#name",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+                "Root your emulator"
+            ]
+        },
+        "command": {
+            "title": "command",
+            "description": "A command of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#command",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+                "adb root"
+            ]
+        },
+        "tags": {
+            "title": "tags",
+            "description": "Tags of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#tags",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "description": "A tag of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#tags",
+                "type": "string",
+                "minLength": 1,
+                "examples": [
+                    "android"
+                ]
+            }
+        },
+        "description": {
+            "title": "description",
+            "description": "A description of the current workflow\nhttps://github.com/EmilySeville7cfg/warp-workflows/blob/main/FORMAT.md#description",
+            "$ref": "#/definitions/nullable-string"
```

Comment:
I'm by no means a JSON schema expert, but could you clarify why we need the `nullable-string` definition here? Why can't we use JSON schema's support for Required properties https://json-schema.org/understanding-json-schema/reference/object.html#required-properties